packages/core/core/src/types.js

```diff
@@ -46,6 +45,7 @@ export type ExtendableParcelConfigPipeline = $ReadOnlyArray<
 >;
 
 export type ProcessedParcelConfig = {|
+  // TODO ProjectPath?
   extends?: PackageName | FilePath | Array<PackageName | FilePath>,
```

Comment:
Haven't determined yet if this is ever an absolute path (and if it's stored in the cache at all)

Reply:
This property was apparently unused, so I just removed it...